director/stencila_open/templates/open/main.html

```diff
@@ -184,11 +194,109 @@ <h5><strong>Github</strong></h5>
+                    How can Stencila Open help?
+                  </span>
+                </header>
+              </div>
+          </div>
+
+          <div class="columns is-vcentered is-centered has-background-white-bis">
+            <div class="column is-3">
+              <img src="{% static 'img/file-conversion-graphic.png' %}" alt="" />
+            </div>
+
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Seamless file conversion
+              </h3>
+
+              <p class="is-size-5">
+              Go back-and-forth across file formats as needed so you can efficiently collaborate
+              with every stakeholder.
+              </p>
+              <p class="is-size-5">
+              Whether you need to work with researchers across teams who are using R Markdown
+              instead of Jupyter Notebooks, or need your paper proofread in Word, Stencila can help
+              facilitate it all.
+              </p>
+
+              <br />
+
+              <p class="is-size-5">
+              <code>/Open</code> supports opening Markdown, R Markdown, Jupyter Notebooks, DOCX (Word), and Google&nbsp;Docs.
+              </p>
+
+              <p class="is-size-5">
+                <a href="//stenci.la/#beta-signup">Sign up for Stencila Beta</a> to convert between even <a
+                   href="https://github.com/stencila/encoda#formats">more formats</a> like JATS and LaTex.
+              </p>
+
+              <hr class="is-small">
+            </div>
+          </div>
+
+
+          <div class="columns is-vcentered is-centered has-background-light">
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Preserve data across formats
+              </h3>
+
+              <p class="is-size-5">
+                By embedding the source-code inside generated figures, we are able to maintain reproducibility
+                throughout the entirety of the authoring lifecycle—even in formats that don’t traditionally support code
+                elements.
+              </p>
+
+              <br />
+
+              <p class="is-size-5">
+                You’ll never have to manually type out tracked changes from Word into your research paper again!
+              </p>
+
+              <hr class="is-small">
+            </div>
+
+            <div class="column is-3">
+              <img src="{% static 'img/file-conversion-lossless.png' %}" alt="" />
+            </div>
+          </div>
+
+
+          <div class="columns is-vcentered is-centered has-background-white-bis">
+            <div class="column is-3">
+              <img src="{% static 'img/file-conversion-themes.png' %}" alt="" />
+            </div>
+
+            <div class="column is-4 is-offset-1">
+              <h3 class="title is-size-4-mobile">
+                Beautifully styled papers to match&nbsp;your&nbsp;needs
```

Comment:
Personally I cringe every time a company describes their products as "beautiful". I find it extremely overused these days.

Reply:
I don't feel quite as strongly as Ben but do also hesitate a little with that wording. Plus, I don't really think the generated HTML  documents are always beautiful - it depends a lot on the content.

Reply:
Mind suggesting an alternative please @beneboy?

Reply:
Don't feel the need to change it just on my account, just my opinion. My first thought was "elegantly styled" but that's in the same vein, like come on… beautiful?, elegant? it's a freakin HTML page!

What about just "Papers style to match your needs"?

Reply:
Thanks for the input everyone, helps guide the changes.
I changed it to `Multiple themes to style your manuscript` to keep it objective and highlight the fact that users have options.